AdminWebsite/AdminWebsite/Contracts/Requests/JudiciaryParticipantRequest.cs

```diff
@@ -1,9 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+
 namespace AdminWebsite.Contracts.Requests;
 
 public class JudiciaryParticipantRequest
 {
     public string PersonalCode { get; set; }
     public string Role { get; set; }
+    
+    [StringLength(255, ErrorMessage = "Display name max length is 255 characters")]
+    [RegularExpression(@"^[\p{L}\p{N}\s',._-]+$")]
```

Comment:
please can you add these rules to bookings API too. The two apps should match

requests to update should be

JudiciaryParticipantRequest
UpdateJudiciaryParticipantRequest

Reply:
done